src/artist-selection/artist-picker.tsx

```diff
@@ -50,11 +52,11 @@ export const ArtistPicker: React.FC<ArtistPickerProps> = observer(({spotifyApi,
             }
         </>
     )
-    //TODO NOAM add a slider to choose the amount of songs you wish to get in the created playlist, and pass as limit to getRecommendations
 });
 
 interface ArtistPickerProps {
     spotifyApi: SpotifyApi;
     selectedGenres: string[];
+    onSliderChange: (e: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
I think "onSliderValueChange" is more descriptive

Reply:
Agree